dns/zone-configs/kubernetes.io.yaml

```diff
@@ -183,3 +183,6 @@ pfjqlwt67ty3.testgrid:
 velodrome:
   type: CNAME
   value: velodrome.k8s.io.
+youtube:
```

Comment:
Why yt.k8s.io and not yt.kubernetes.io ?

Reply:
and why not go.k8s.io/youtube ?

Reply:
The answer to both is fewer characters, easier to remember. I can add them if you'd like but, I don't know if they'd be utilized in lieu of the others.